### DISCUSSION TITLE
maven central artifact for 5.2.2 join faces.maven-plugin missing?

### DISCUSSION THREAD

**Discussion by StefanSchubert** [joinfaces/joinfaces#2172] [Q&A] [answered]
Hi, Im trying to patch my app to join faces 5.2.2

While 

<groupId>org.joinfaces</groupId>
<artifactId>joinfaces-dependencies</artifactId>
<version>${joinfaces.version}</version>

and 

<groupId>org.joinfaces</groupId>
<artifactId>primefaces-spring-boot-starter</artifactId>
<version>${joinfaces.version}</version>

can be resolved with 5.2.2

I got complaints on this one:

                    <groupId>org.joinfaces</groupId>
                    <artifactId>joinfaces-maven-plugin</artifactId>
                    <version>${joinfaces.version}</version>

The latest one I can see there:

https://mvnrepository.com/artifact/org.joinfaces/joinfaces-maven-plugin
is 5.2.0-m3

Is it OK to mix these versions?

**Comment by StefanSchubert** [comment]
Tried it with the 5.2.0-m3 only for the maven plugin. Seems to work so far....

**Comment by larsgrefer** [comment, marked as answer]
We removed the maven-plugin with JoinFaces 5.2.x. as we now integrate with spring-aot.

Have a look at the updated reference documentation: https://docs.joinfaces.org/current/reference/#_maven

More background can be found here: https://github.com/joinfaces/joinfaces/discussions/1886

**Reply by StefanSchubert** [reply]
THX!
